server/services/tuya/index.js

```
@@ -6,6 +6,98 @@ const { STATUS } = require('./lib/utils/tuya.constants');
+      return true;
+    }
+  }
+
+  /**
+   * @description Clear pending quick reconnect timers and reset state.
+   * @example
+   * clearQuickReconnects();
+   */
+  function clearQuickReconnects() {
+    if (quickReconnectTimeouts.length > 0) {
+      quickReconnectTimeouts.forEach((timeoutId) => clearTimeout(timeoutId));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)